.editorconfig

```diff
@@ -0,0 +1,14 @@
+# EditorConfig helps developers define and maintain consistent coding styles between different editors and IDEs
+# @see http://editorconfig.org
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = spaces
+tab_width = 2
+
+[{*.{rb,erb,js,coffee,json,yml,css,scss,sh,markdown,md,html}]
+indent_size = 2
```

Comment:
This seems like an odd way to set the indent. Why not set `indent_size = 2` for everything, and then set a different indent size for anything that doesn't fit that?

Reply:
Makes sense. I will update accordingly.